src/Model/Image.php

```diff
@@ -110,109 +287,120 @@ public function DuoTone($darkColour = '', $lightColour = null)
             $duoTone = $darkColour;
         }
 
-        $transformation = [];
-
-        if (array_key_exists('transformation', $this->options)) {
-            $transformation = $this->options['transformation'];
-        }
-
-        $this->options['transformation'] = array_merge($transformation, [
-            ['effect' => 'grayscale'],
-            ['effect' => $duoTone],
-        ]);
-
-        return $this;
+        return $this
+            ->Effect('grayscale')
+            ->Effect($duoTone);
     }
 
-    public function Brightness($value = 0)
+    /**
+     * Gets the Cloudinary URL for the image at the requested size, crop etc.
+     * NOTE: Uses the `is_bool` check because SS template passes `true`/`false` as string so uses json_decode
+     * @param Int $width
+     * @param Int $height
+     * @param String $crop
+     * @param String $quality
+     * @param String $gravity
+     * @param Boolean $fetchFormat
+     */
+    public function URL($width = 100, $height = 100, $crop = 'fill', $quality = 'auto', $gravity = 'auto', $fetchFormat = 'auto')
     {
-        if (is_null($value) || !(int) $value) {
-            return $this;
-        }
+        return $this
+            ->Size($width, $height)
+            ->Crop($crop)
+            ->Quality($quality)
+            ->Gravity($gravity)
+            ->FetchFormat($fetchFormat);
+    }
 
-        $transformation = [];
+    /**
+     * @return string
+     */
+    public function forTemplate()
+    {
+        $transformations = [];
 
         if (array_key_exists('transformation', $this->options)) {
-            $transformation = $this->options['transformation'];
+            $transformations = $this->options['transformation'];
         }
 
-        $this->options['transformation'] = array_merge($transformation, [
-            ['effect' => 'brightness:' . $value],
-        ]);
+        if (array_key_exists('resource_type', $transformations[0]) === false) {
+            $transformations[0]['resource_type'] = $this->ResourceType;
+        }
 
-        return $this;
-    }
+        if (array_key_exists('type', $transformations[0]) === false) {
+            $transformations[0]['type'] = $this->Type;
+        }
 
-    public function Effect($effect, $value)
-    {
-        if (is_null($effect) || !strlen($effect)) {
-            return $this;
+        if ($this->ImageLink()->exists()) {
+            if (array_key_exists('gravity', $transformations[0]) === false || $transformations[0]['gravity'] === 'auto') {
```

Comment:
`if (!array_key_exists('gravity', $transformations[0])) {`
The gravity is set to `auto` at the beginning